Musical Collection/Musical Collection Console App/Interfaces/Classes Interfaces/IPlaylist.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Musical_Collection_Console_App.Classes;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Musical_Collection_Console_App.Interfaces.Classes_Interfaces
+{
+    public interface IPlaylist : IEntity
+    {
+        string CreatorName { get; set; }
```

Comment:
Creator name should have only getter. Do we want to change the playlist creator ?